shared/src/persistence/elasticsearch/fetchPendingItems.js

```diff
@@ -1,19 +1,24 @@
+const { isCodeEnabled } = require('../../../../codeToggles');
 const { search } = require('./searchClient');
 
-exports.fetchPendingItems = async ({ applicationContext, judge, page }) => {
-  const caseSource = [
-    'associatedJudge',
-    'caseCaption',
-    'docketNumber',
-    'docketNumberSuffix',
-    'status',
-  ];
-  const docketEntrySource = [
-    'docketEntryId',
-    'documentType',
-    'documentTitle',
-    'receivedAt',
-  ];
+exports.fetchPendingItems = async ({
+  applicationContext,
+  judge,
+  page,
+  source,
+}) => {
+  const caseSource = isCodeEnabled(7134)
```

Comment:
This toggle appears to be keeping the same functionality as was there previously when the toggle is set to 'true'. Is that the intention?

Reply:
This file already had a toggle when we implemented the bug fix for 7134. 

The first bug we fixed, we implemented the toggle using old/new files and switching the method that was called instead via applicationContext. When we realized we had to implement another code toggle that touched this file, we removed the .old and had to reimplement the toggle for the previous bug fix in this way. I can totally understand how this looks confusing, it may be easier to discuss in a Zoom if you have time!

Reply:
I think I understand - thanks!